### PR TITLE
fix(privatedb.order): warn when a free offer is still available

### PR DESCRIPTION
**Pull request by euhmeuh**
MANAGER-1281

NB: This controller has Bad™ code. It has been refactored in Cloud manager and I'm planning to remove all this mess with the new code from cloud universe. In the mean time, please be cautious before reading this code. You might experience dizziness, uncomfortable itches, and even vomiting. Thank you for your understanding.

NB2: I couldn't stop from renaming some really bad stuff anyways to ease the future refactoring.

### CODE REVIEW

File: client/app/private-database/order/sql-database-order.html [hunk 1]
```diff
@@ -100,52 +100,60 @@ <h2 class="mt-5" data-translate="privateDatabase_order_sql_customize_title"></h2
...
                              data-translate="privateDatabase_order_step1_no_hosting_warning"
-                             data-ng-if="sqlOrderCtrl.isPrivateDb() && sqlOrderCtrl.model.hosting === sqlOrderCtrl.noHostValue">
+                             data-ng-if="$ctrl.isPrivateDb() && $ctrl.model.hosting === $ctrl.noHostValue">
                         </div>
                     </div>
 
-                    <div class="loader mt-5 text-center" data-ng-if="sqlOrderCtrl.loading.durations"></div>
+                    <div class="alert alert-warning" role="alert"
+                         data-ng-if="$ctrl.isPrivateDb() && $ctrl.selectedHosting && $ctrl.selectedHosting.stillHasFreeDbOffer">
+                        <h3 class="h5" data-translate="privateDatabase_order_sql_included_title"></h3>
```
Comment (FredericEspiau) [inline client/app/private-database/order/sql-database-order.html]: https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/ovh-ui-kit/typography

File: client/app/private-database/order/sql-database-order.html [hunk 2]
```diff
@@ -100,52 +100,60 @@ <h2 class="mt-5" data-translate="privateDatabase_order_sql_customize_title"></h2
...
                         </div>
                         <div class="alert alert-warning" role="alert"
                              data-translate="privateDatabase_order_step1_no_hosting_warning"
-                             data-ng-if="sqlOrderCtrl.isPrivateDb() && sqlOrderCtrl.model.hosting === sqlOrderCtrl.noHostValue">
+                             data-ng-if="$ctrl.isPrivateDb() && $ctrl.model.hosting === $ctrl.noHostValue">
                         </div>
                     </div>
 
-                    <div class="loader mt-5 text-center" data-ng-if="sqlOrderCtrl.loading.durations"></div>
+                    <div class="alert alert-warning" role="alert"
```
Comment (FredericEspiau) [inline client/app/private-database/order/sql-database-order.html]: `oui-message`

File: client/app/private-database/translations/Messages_fr_FR.xml [hunk 3]
```diff
@@ -512,8 +512,8 @@
    <translation id="privateDatabase_order_sql_domains_pack_db_SQLPERSO_5_BASES_200_MB" qtlid="352282">5 bases de 200 Mo</translation>
    <translation id="privateDatabase_order_sql_domains_pack_db_SQLPERSO_5_BASES_400_MB" qtlid="352295">5 bases de 400 Mo</translation>
    <translation id="privateDatabase_order_sql_domains_pack_db_SQLPERSO_5_BASES_800_MB" qtlid="352308">5 bases de 800 Mo</translation>
-   <translation id="privateDatabase_order_sql_included_title" qtlid="26551">Informations</translation>
-   <translation id="privateDatabase_order_sql_included_info" qtlid="382167">Service inclus dans votre offre {1}. Pour l'activer, rendez-vous dans l'onglet <a href="{0}">informations générales de votre hébergement</a>.</translation>
+   <translation id="privateDatabase_order_sql_included_title">Attention !</translation>
+   <translation id="privateDatabase_order_sql_included_info">N'oubliez pas que vous avez déjà une base de données incluse dans votre offre <strong>{0}</strong>. Pour l'activer, rendez-vous dans l'onglet <a href="{1}">Informations Générales de votre hébergement</a>.</translation>
```
Comment (FredericEspiau) [inline client/app/private-database/translations/Messages_fr_FR.xml]: Extract `strong` and `a`

File: client/app/private-database/order/sql-database-order.html (hunk 1 above)
Comment (antleblanc) [reply client/app/private-database/order/sql-database-order.html]: `oui-header_5` is available.

https://github.com/ovh-ux/ovh-ui-kit/pull/296/files#diff-bc7d9dc2b0a922de7ac1252784ab8615R37

**Pull request merged**